documentation/common/attributes.adoc

```diff
@@ -32,6 +33,10 @@
 //kafka Exporter links
 :kafka-exporter-project: link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter^]
 
+//OAuth attributes and links
+:external-type: loadbalancer
```

Comment:
I'm not sure we really need this as variable. I think we can just hardcode it.

Reply:
Okay. We can stick with loadbalancer.